### ISSUE TITLE
Provide per alias signatures

### ISSUE THREAD

**Issue by taajo** [jeevatkm/ReplyWithHeader#77]
In Mac Mail you can set up multiple aliases for an account by listing multiple email addresses in "Preferences > Accounts > your_account >Email Address:", for example "user1@me.com, user2@me.com, ..."  When you send an email then, you can pick which alias the email is coming from.  However neither Mail nor RWH lets you specify a signature to use per alias.  It would be very useful to have the ability in RWH > Manage Signatures to assign a default signature to not just for an account as available now but for each alias for each account.

Thank you.

**Comment by jeevatkm** [comment]
@taajo I will consider this as a feature request. Thanks for bring it up!

**Comment by jeevatkm** [comment]
Did deep analysis to get hang around email aliases in the Mail.app. It seems internal APIs are strange enough to hide details :)

Still try out! let you know if somethings pops up.

**Comment by ziggyke** [comment]
I don't think this is possible or allowed by Apple. Because the function isn't (yet) built into mail.app
